package.json

```diff
@@ -32,7 +32,8 @@
     "lint:examples": "eslint ./examples/**/*.js",
     "lint:examples:fix": "eslint ./examples/**/*.js --fix",
     "lint:markdown": "./node_modules/.bin/markdownlint $(git ls-files '*.md') -i ./CHANGELOG.md",
-    "lint:markdown:fix": "./node_modules/.bin/markdownlint $(git ls-files '*.md') -i ./CHANGELOG.md --fix"
+    "lint:markdown:fix": "./node_modules/.bin/markdownlint $(git ls-files '*.md') -i ./CHANGELOG.md --fix",
+    "reset": "lerna clean -y && rm -rf node_modules && git clean -fdx && npm i && npm run compile && npm run lint:fix"
```

Comment:
`git clean -fdx` this is bad, you can loose all your temporary work, please remove it

Reply:
done